graphics/dvisvgm/Portfile

```diff
@@ -13,24 +14,23 @@ description         DVI to SVG converter
 long_description    The command-line utility dvisvgm is a tool for TeX/LaTeX users. It \
                     converts DVI files to the XML-based scalable vector graphics format SVG.
 
-github.setup        mgieseki dvisvgm 2.3.4
-revision            4
-checksums           rmd160 a6b6bae339fb915b2c5f585b4f176a93fd901fc3 \
-                    sha256 5399f9ac2b254bc8a5c4862bc2126f2e8c6caec243c3205c0223564371100129 \
-                    size   2896641
-
 github.tarball_from releases
 
-homepage            http://dvisvgm.bplaced.net/
+checksums           rmd160  bb854e937a58b5016b0dfc8db61ba22abcaa35a2 \
+                    sha256  bda5e875f810be08e4d26add9c4ea191f53aa9ae14cd2c1fa130e0c8922ab0c4 \
+                    size    2911596
 
 depends_build       port:asciidoc \
                     port:pkgconfig
 
-depends_lib         port:freetype \
+depends_lib         port:brotli \
+                    port:freetype \
                     port:ghostscript \
                     port:potrace \
                     port:texlive-bin \
+                    port:woff2 \
                     port:zlib
+                    # port:xxhash
```

Comment:
Not sure what this bit is for? Perhaps you should comment the comment?

Reply:
This is a dependency which we are not packaging yet, so it uses the library shipped with the sources.

Reply:
Right, but that could have been mentioned in a comment in the portfile.

Reply:
What's wrong with the (non-outdated version of) the current comment?

Reply:
I got confused by the github interface and didn't see that it had changed. Apologies.